app/src/main/res/values-tr/strings.xml

```diff
@@ -35,72 +35,72 @@
+    <string name="preferences_note_deletion_time_month">30 gün sonra</string>
 
     <string name="preferences_header_backup">Yedekleme</string>
     <string name="preferences_create_backup">Yedek oluştur</string>
-    <string name="preferences_create_backup_subtext">Tüm notları yedek dosyasına aktar.</string>
-    <string name="preferences_restore">Geri yükle</string>
-    <string name="preferences_restore_subtext">Notları yedek dosyadan yükle.</string>
+    <string name="preferences_create_backup_subtext">Tüm notları bir dosyaya yedekler.</string>
+    <string name="preferences_restore">Yedekten geri yükle</string>
+    <string name="preferences_restore_subtext">Notları bir yedek dosyadan geri yükler.</string>
     <string name="preferences_backup_strategy">Ekler için yedekleme yöntemi</string>
-    <string name="preferences_backup_strategy_include_files">Herşeyi yedekle</string>
-    <string name="preferences_backup_strategy_keep_info">Sadece açıklamayı ve yerel yolu yedekle</string>
-    <string name="preferences_backup_strategy_keep_nothing">Ekleri yedekleme</string>
+    <string name="preferences_backup_strategy_include_files">Her şeyi yedekle</string>
+    <string name="preferences_backup_strategy_keep_info">Sadece yedekleme açıklamasını ve yerel konumu yedekle</string>
+    <string name="preferences_backup_strategy_keep_nothing">Ekler yedeklenmesin</string>
 
     <string name="preferences_header_syncing">Senkronizasyon</string>
-    <string name="preferences_go_to_sync_settings">Senkronize ayarlarına git</string>
-    <string name="preferences_currently_syncing_with">Şu anda %s senkronize edilmiyor</string>
+    <string name="preferences_go_to_sync_settings">Senkronizasyon ayarlarına git</string>
+    <string name="preferences_currently_syncing_with">Şu anda %s ile senkronize ediliyor</string>
     <string name="preferences_currently_not_syncing">Şu anda senkronize edilmiyor</string>
-    <string name="preferences_cloud_service">Senkronize hizmeti</string>
-    <string name="preferences_cloud_service_disabled">Kapalı</string>
+    <string name="preferences_cloud_service">Senkronizasyon hizmeti</string>
+    <string name="preferences_cloud_service_disabled">Devre dışı</string>
     <string name="preferences_cloud_service_nextcloud">Nextcloud</string>
-    <string name="preferences_sync_experimental_warning">Senkronize şu anda tamamlanmamış.\nHatalarla karşı karşıya gelebilirsin.</string>
-    <string name="preferences_nextcloud_limited_features">Nextcloud Notları etiket, eklenti, hatırlatıcı, görev listesi vb. özellikleri desteklemiyor.</string>
+    <string name="preferences_sync_experimental_warning">Senkronizasyon işlevi şimdilik deneyseldir.\nHatalarla karşılaşabilirsiniz.</string>
+    <string name="preferences_nextcloud_limited_features">Lütfen Nextcloud Notes'un etiketler, ekler, hatırlatıcılar, görevler gibi özellikleri desteklemediğini bilerek ilerleyin.</string>
```

Comment:
I think it's failing because of the un-escaped `'` in `Notes'un`

Reply:
I can not believe Github thought that it needed to be escaped.